mlir/utils/performance/perfRunner.py

```diff
@@ -793,6 +822,127 @@ def benchmarkMIOpenWithMLIRKernels(configs, arch, filename, paths: Paths):
     chip = GFX_CHIP_RE.search(arch).group(0)
     df.to_csv(chip + '_' + filename, index=False)
 
+def findRunCommand(filename, includes=['RUN']):
+    with open(filename, 'r') as f:
+        for line in f:
+            if line.strip().startswith("//") and not 'DISABLE' in line and all(word in line for word in includes):
+                command = line.split("RUN")[1].strip()[2:] # Remove the "//" and "RUN" prefixes and leading/trailing whitespace
+                parts = command.split('|')  # Split the command using the "|" separator
+                return parts[0].strip()
+                # Stop processing lines after finding the first matching line
+    # Not a valid test
+    print("WARNING: cannot find RUN command in the test file")
+    return ""
+
+def getTestVector(file_path, paths:Paths):
+    firstCommand = findRunCommand(file_path)
+    if (not firstCommand):
+        return ""
+
+    if ("-migraphx-to-tosa" in firstCommand):
+        rocmlirOptCommand = [paths.mlir_paths.rocmlir_opt_path, '-migraphx-to-tosa', file_path]
+        rocmlirDriverCommand = [paths.mlir_paths.rocmlir_driver_path, '-host-pipeline', 'partition,highlevel', '-targets', getChip()]
```

Comment:
Strong opinion: Add `,runner` to the host pipeline, and then, instead of using `xmir-runner`, use `mlir-cpu-runner`. This makes absolutely sure we don't benchmark GPU compilations or the like.

Reply:
I tried to use mlir-cpu-runner. For some unknown reason, rocprof didn't generate result.stats.csv without giving any error messages.

Reply:
Did you pass in `runner` as part of the `rocmlir-driver` host pipeline? Also, if you printed out the commands and ran them manually, did you get reasonable results?

Reply:
Yes, I did pass in runner and run the commands manually. The tests were successful. But rocprof didn't generate the file result.stats.csv.

Reply:
If you use the runner pipeline, you should be able to use `mlir-cpu-runner -O2` like all the other perf tests

Reply:
For some reason that I don't understand, rocprof doesn't generate results.stats.csv if I use cpu-runner here.